Source/Door_4D4.hpp

```diff
@@ -8,6 +9,34 @@ class Door_10;
 class Sprite_4C;
 class Ped;
 
+namespace door_open_type
+{
+enum
+{
+    unknown1 = 0,
+    any_player = 1,
+    any_car = 2,
+    one_car = 3,
+    one_model = 4,
+    one_char_on_foot = 5,
+    any_player_one_car = 6,
+};
+//static_assert(sizeof(door_open_type) == 4);
```

Comment:
we do have a macro called `GTA2_ASSERT_SIZEOF_ALWAYS` which works on both VC6 and MSVC2022 fyi